src/org/hammerhead226/sharkmacro/motionprofiles/Recording.java

```diff
@@ -116,4 +133,30 @@ private boolean areEqual(int idx, double comparator) {
 		return (recordings.get(0).get(idx) == comparator && recordings.get(1).get(idx) == comparator
 				&& recordings.get(2).get(idx) == comparator && recordings.get(3).get(idx) == comparator);
 	}
+
+	/**
+	 * Converts raw units into wheel rotations.
+	 * 
+	 * @param rawUnits
+	 *            raw sensor units
+	 * @param sensorUnitsPerRotation
+	 *            amount of raw units per wheel rotation
+	 * @return raw units as wheel rotations
+	 */
+	private double toRotations(double rawUnits, double sensorUnitsPerRotation) {
+		return rawUnits * (1.0 / sensorUnitsPerRotation);
+	}
```

Comment:
Would it be useful to provide a `sensorUnitsPerRotation = 4096` version of the method?